docs/algorand/for-end-users/Algorand-Standard-Assets-support.mdx

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_position: 5
+---
+import asadispenser from "@site/static/img/asadispenser.png";
+import addasset from "@site/static/img/addasset.png";
+import EUReinwallet from "@site/static/img/EUReinwallet.png";
+import addingEuretowallet from "@site/static/img/addingEuretowallet.png";
+import EURewrappingchoice from "@site/static/img/EURewrappingchoice.png";
+import EURewrappingconfirmation from "@site/static/img/EURewrappingconfirmation.png";
+import ASAwrappedconfirmations from "@site/static/img/ASAwrappedconfirmations.png";
+import AddASSAtoMetamask from "@site/static/img/AddASSAtoMetamask.png";
+import ChooseASAtounwrap from "@site/static/img/ChooseASAtounwrap.png";
+import ConfirmASAunwrapping from "@site/static/img/ConfirmASAunwrapping.png";
+import tokenerc20allowance from "@site/static/img/tokenerc20allowance.png";
+import erc20allowance from "@site/static/img/erc20allowance.png";
+import unwrappingreqconfirmation from "@site/static/img/unwrappingreqconfirmation.png";
+
+
+# Algorand Standard Assets support 
+<p style={{textAlign:"justify"}}>
+This section gives brief introduction about Algorand Standard Assets (ASAs) supported by the Milkomeda A1 Rollup Bridge and how to move ASA (wrap or unwrap) between Milkomeda A1 Rollup and mainchain (Algorand).
```

Comment:
This section gives a brief introduction of the Algorand Standard Assets (ASAs) supported by the Milkomeda A1 Rollup Bridge and how to move ASAs (wrapping or unwrapping) between the Milkomeda A1 Rollup and the mainchain (Algorand).